File: src/main/java/itmo/is/project/mapper/user/AccountMapper.java
package itmo.is.project.mapper.user;

import itmo.is.project.dto.user.AccountDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.model.user.Account;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring", uses = {UserMapper.class})
public interface AccountMapper extends EntityMapper<AccountDto, Account> {
}

File: src/main/kotlin/itmo/isproject/mapper/user/AccountMapper.kt
package itmo.isproject.mapper.user

import itmo.isproject.dto.user.AccountDto
import itmo.isproject.mapper.EntityMapper
import itmo.isproject.model.user.Account
import org.mapstruct.Mapper

@Mapper(uses = [UserMapper::class])
interface AccountMapper : EntityMapper<AccountDto, Account>
